modules/dnn/src/op_webnn.hpp

```diff
@@ -31,6 +31,66 @@ constexpr bool haveWebNN() {
 #endif
 }
 
+#ifdef HAVE_WEBNN
+
+class WebNNBackendNode;
+
+
+class WebNNGraph
+{
+public:
+    WebNNGraph();
+
+    bool isInitialized();
+
+    void init(Target targetId);
+
+    void forward(const std::vector<Ptr<BackendWrapper> >& outBlobsWrappers);
+
+    void createGraph(Target targetId);
+
+private:
+    ml::GraphBuilder builder;
+    ml::Context mContext;
+    ml::Graph mGraph;
+    ml::NamedResults mResults;
```

Comment:
If it is for graph building, it should be ml::NamedOperands

Reply:
Fixed.